draft/spec/index.html

```diff
@@ -1031,6 +1031,43 @@ <h2>Versioned OCFL Object</h2>
 </pre>
     </section>
 
+    <section id="example-object-with-file-renaming">
+        <h2>File Renaming in an OCFL Object</h2>
+        <p>
+            The following example OCFL Object inventory shows how exiting file paths may differ from
```

Comment:
`exiting [sic] file paths` => `storage paths`

Reply:
My understanding is that "existing path" is now "content path".
https://github.com/OCFL/spec/issues/313

Reply:
I thought it best to stick with the current terminology until that is changed everywhere in response to #313